opm/parser/eclipse/EclipseState/Tables/TableManager.cpp

```diff
@@ -337,8 +333,7 @@ namespace Opm {
         auto& container = forceGetTables(keywordName , numTables);
         for (size_t tableIdx = 0; tableIdx < keyword->size(); ++tableIdx) {
             const auto tableRecord = keyword->getRecord( tableIdx );
-            std::shared_ptr<PlyrockTable> table = std::make_shared<PlyrockTable>();
-            table->init( tableRecord );
+            std::shared_ptr<PlyrockTable> table = std::make_shared<PlyrockTable>(tableRecord);
```

Comment:
Well - the concept of defaulted tables *should* be handled by the `TableContainer` class.